### PR TITLE
Plugin autoupdate

### PR DESCRIPTION
**Pull request by domi4484**

### CODE REVIEW

**Comment by domi4484** [comment]
@m-kuhn implementation is working but the plugin manager should have a better separation logic/gui to simplify things.  I would like to do it if we have some spare time

File: kadas/app/kadaspluginmanager.h [hunk 1]
```diff
@@ -61,6 +64,9 @@ class KadasPluginManager: public KadasBottomBar, private Ui::KadasPluginManagerB
 
   private:
 
+    const int INSTALLED_TREEWIDGET_COLUMN_NAME = 0;
+    const int INSTALLED_TREEWIDGET_COLUMN_VERSION = 1;
```
Comment (m-kuhn) [inline kadas/app/kadaspluginmanager.h:68]: Thanks !

**Review by m-kuhn — APPROVED** [review]
LGTM, thanks.
Can you open a new ticket for the cleanup task so we can prioritize with @bernspi ?

We should probably start to mandate running `./scripts/formatSourceFiles.sh` for code style

**Comment by domi4484** [comment]
> We should probably start to mandate running `./scripts/formatSourceFiles.sh` for code style

For me astyle complains about invalid options. Should we either add a pre-commit configuration?

**Pull request merged**